servers/rendering/renderer_rd/environment/fog.cpp

```diff
@@ -966,8 +966,8 @@ void Fog::volumetric_fog_update(const VolumetricFogSettings &p_settings, const P
 
 	VolumetricFogShader::ParamsUBO params;
 
-	Vector2 frustum_near_size = p_cam_projection.get_viewport_half_extents();
-	Vector2 frustum_far_size = p_cam_projection.get_far_plane_half_extents();
+	Vector2 frustum_near_size = p_cam_projection.get_viewport_half_extents() * 2.0;
+	Vector2 frustum_far_size = p_cam_projection.get_far_plane_half_extents() * 2.0;
```

Comment:
This seems odd, you are double the near and far size but not making an equivalent change in the shader, so this is likely to change behaviour. Likely needs to be reverted

Reply:
Yeah, I think I overlooked it. Thanks!